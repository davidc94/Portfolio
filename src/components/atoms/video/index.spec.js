import React from 'react';
import renderer from 'react-test-renderer';
import StyledVideo from './index';
import Colors from '../../../utils/colors';

describe('StyledVideo component', () => {
  it('render default StyledVideo as expected', () => {
    const render = renderer.create(
      <StyledVideo />,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});