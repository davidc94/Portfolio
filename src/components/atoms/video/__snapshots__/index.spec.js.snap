// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledVideo component render default StyledVideo as expected 1`] = `
<video
  autoPlay={true}
  className="sc-bdVaJa gDlMSY"
  loop={true}
  muted={true}
  poster=""
  preload="auto"
>
  <source
    src=""
    type="video/mp4"
  />
</video>
`;
